examples/hosts/app-integration/external-controller/package.json

```diff
@@ -33,22 +33,15 @@
     "webpack:dev": "webpack --env.development"
   },
   "dependencies": {
-    "@fluid-experimental/data-objects": "^0.40.0",
-    "@fluid-experimental/fluid-static": "^0.40.0",
+    "fluid-framework": "^0.40.0",
```

Comment:
Dropping the scope entirely here may lead to some interesting behavior with how we decide which feed to use for the packages.

In .npmrc, the scopes are used to pick the appropriate feed for the alias
https://github.com/microsoft/FluidFramework/blob/c529d698c460fe93c97546d8bc4748eb0d72eb9e/tools/pipelines/templates/include-publish-npm-package-steps.yml#L67

Perhaps we want to preserve the alias here and append the name afterwards?

Reply:
yeah, i'll return the scope, but change the package to simply "fluid-framework"....i step closer to the final name.